com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/TextureUtil.java

```diff
@@ -214,8 +217,19 @@ public static TextureProfile getTextureProfileByPath(TextureProfiles textureProf
         return null;
     }
 
-    public static TextureImage createCombinedTextureImage(TextureImage[] textures, TextureImage.Type type) throws TextureGeneratorException {
-        int numTextures = textures.length;
+    private static int getTextureGenerateResultByteIndex(TextureImage textureImage, int alternative, int mipMap) throws TextureGeneratorException {
```

Comment:
This can probably be written differently, but I didn't understand the old code so I opted for the simplest solution.

Reply:
Yeah, I want the "alternatives" to be removed.
It's not the way we want to provide alternative texture packs imo.